polymorphic/admin/parentadmin.py

```diff
@@ -50,6 +51,7 @@ class PolymorphicParentModelAdmin(admin.ModelAdmin):
 
     #: The child models that should be displayed
     child_models = None
+    exclude_childs = None
```

Comment:
```suggestion
    exclude_children = None
```
(and docs again, please?)

Reply:
Same as before, why the change?